go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/cometbft/cometbft
 
-go 1.22.3
+go 1.22.4
```

Comment:
Similar feedback to here: https://github.com/celestiaorg/celestia-core/pull/1379#discussion_r1628223218

Reply:
It appears that there is consensus to upgrade the Go version.